utils/src/ooinstall/oo_config.py

```diff
@@ -36,19 +36,24 @@ def __init__(self, **kwargs):
         self.public_ip = kwargs.get('public_ip', None)
         self.public_hostname = kwargs.get('public_hostname', None)
         self.connect_to = kwargs.get('connect_to', None)
+        self.run_on = kwargs.get('run_on', None)
 
         # Should this host run as an OpenShift master:
         self.master = kwargs.get('master', False)
 
         # Should this host run as an OpenShift node:
         self.node = kwargs.get('node', False)
+
+        # Should this host run as an HAProxy:
+        self.ha_proxy = kwargs.get('ha_proxy', False)
```

Comment:
Too generic you think? If that could be confused with some other meaning today or in future then yeah maybe we should rename. master_lb looks best to me of those, not something users should normally have to write by hand hopefully. Any other ideas on the name? Just 'loadbalancer'?